integration_tests/test_terraform/tf_state.cfn/templates/tf_state.yaml

```diff
@@ -0,0 +1,73 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Conditions:
+  BucketNameOmitted: !Or
+    - !Equals
+      - !Ref 'BucketName'
+      - ''
+    - !Equals
+      - !Ref 'BucketName'
+      - undefined
+  TableNameOmitted: !Or
+    - !Equals
+      - !Ref 'TableName'
+      - ''
+    - !Equals
+      - !Ref 'TableName'
+      - undefined
+Description: Create Terraform backend components - 1.19.0
```

Comment:
We can drop the "version number" from this description now

Reply:
Fixed.